k8s/templates/deployment.cms.yaml

```
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-cms
```

I would just call this deployment `app`, as it is the landing page itself. I would also rename the deployment and service files, as well as the key in the `values.yaml`.
```suggestion
  name: {{ .Release.Name }}-app
```